src/abagnale.js

```diff
@@ -3,6 +3,20 @@ import slug from 'slug';
 
 slug.defaults.mode = 'rfc3986';
 
+// A safe way to slugify values. If the input is null, undefined, or
+// some other error happens downstream, we simply return `unknown`.
+function safeSlug(value) {
+  let sluggified;
+
+  try {
+    sluggified = slug(value);
+  } catch (err) {
+    sluggified = 'unknown';
```

Comment:
My only question is what does this `unknown` string do? Does it show up in the results anywhere?